Automattic/jetpack
# Forms: Update child blocks transforms

## Proposed changes:

* Update Forms child blocks to allow any transformation between the blocks

### Other information:


## Jetpack product discussion
p8oabR-18y-p2#comment-7173

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a post, add a Form block, and select a template
* Select a child block and use the `Transform To` option to transform the current block into other blocks
* You should be able to transform a child block into any other Form child block
